army schema work/1 table.sql

```diff
@@ -0,0 +1,42 @@
+use ArmyDB
+go
+
+drop table if exists dbo.Soldier
+go 
+
+create table dbo.Soldier
+(
+FirstName VARCHAR(20) not null
+    CONSTRAINT ck_Soldier_FirstName_cannot_be_blank CHECK (FirstName <> ''),
+LastName VARCHAR(20) not null
+    CONSTRAINT ck_Soldier_LastName_cannot_be_blank CHECK (LastName <> ''),
+SSN VARCHAR(10) not null 
+    CONSTRAINT u_Soldier_SSN_must_be_unique unique,
+    CONSTRAINT ck_Soldier_SSN_have_9_digits CHECK (SSN like '%%%%%%%%-%'),
```

Comment:
-- LL this constraint should check that the nine digits are only numbers and not any other characters (besides for the dash towards the end)